book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'rental'
+
+# Book class created with title and author attribute
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    @rentals << rental
```

Comment:
This line is not necessary since the creation of the rental takes care of adding it to book's `@rentals` list (`line 9` in the `rental.rb` file).